ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -156,35 +149,57 @@ - (void)setError:(NSError * __nullable)error {
 const CGFloat CONTINUE_ALPHA_OPAQUE = 0;
 
 - (void)updateConstraints {
+    if (_variableConstraints) {
```

Comment:
Yes, I'm aware we're doing it elsewhere, but I recalled discussing that exact line a few weeks back. But yes, that's is a good interpretation - the documentation is just saying you should avoid creating an infinite recursion. Ignore my comment.